app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,261 @@
+        "Höhe so schnell wie möglich. \n\n"
+
+        "Bauteilerhöhungen. \n\n"
+
+        Die Ruhehöhe für einen Komponententyp ist für alle Apps gleich (z. B. die FAB-Höhe).
+        msgstr "variiert nicht von 6dp in einer App bis 16dp in einer anderen App. \n"
+        "Komponenten können je nach Tiefe auf verschiedenen Plattformen unterschiedliche Aufstandshöhen haben."
+        "der Umgebung (z. B. TV hat eine größere Tiefe als mobile oder Desktop). \n\n"
+
+        "Responsive Elevation und dynamische Elevation Offsets. \n\n"
+
+        "Einige Komponententypen haben eine ansprechende Höhe, dh sie ändern die Höhe in Abhängigkeit von der Höhe."
+        für Benutzereingaben (z. B. normale, fokussierte und gedrückte) oder Systemereignisse.
+        msgstr "Änderungen werden konsistent mit dynamischen Höhenversätzen implementiert. \n"
+        "Dynamische Höhenversätze sind die Zielhöhe, auf die sich eine Komponente relativ zubewegt."
+        "In den Ruhezustand der Komponente. Sie stellen sicher, dass die Höhenänderungen konsistent sind."
+        "über Aktionen und Komponententypen hinweg. Zum Beispiel haben alle Komponenten, die auf der Presse angehoben werden,
+        msgstr "die gleiche Höhenänderung relativ zu ihrer Ruhehöhe. \n"
+        "Sobald das Eingabeereignis abgeschlossen oder abgebrochen ist, kehrt die Komponente in den Ruhezustand zurück."
+        "Höhe. \n\n"
+
+        "Höhenstörungen vermeiden. \n\n"
+
+        "Komponenten mit reaktionsschnellen Höhen können auf andere Komponenten treffen, wenn sie sich bewegen"
+        "ihre Ruhehöhen und dynamischen Höhenversätze. Weil Material nicht passieren kann"
+        "Durch andere Materialien vermeiden die Komponenten auf vielfältige Weise, dass sie sich gegenseitig stören."
+        "Ob auf Komponentenbasis oder unter Verwendung des gesamten App-Layouts. \n"
+        "Auf Komponentenebene können Komponenten bewegt oder entfernt werden, bevor sie Störungen verursachen."
+        Beispielsweise kann eine schwebende Aktionstaste (Floating Action Button, FAB) verschwinden oder den Bildschirm verlassen, bevor ein
+        msgstr "Benutzer nimmt eine Karte in die Hand oder sie kann sich bewegen, wenn eine Snackbar erscheint. \n"
+        "Entwerfen Sie auf Layoutebene Ihr App-Layout, um die Möglichkeit von Interferenzen zu minimieren."
+        "Positionieren Sie den FAB beispielsweise an einer Seite des Kartenstroms, damit der FAB nicht stört."
+        "Wenn ein Benutzer versucht, eine der Karten aufzuheben. \n\n"
+    </string>
+    <string name="image">Kamera</string>
+    <string-array name="front">Hallo @array/pref_example_list_values</string-array>
+
+    <string name="title_activity_setup">Konfiguration</string>
+    <string name="prompt_email">Email</string>
+    <string name="prompt_password">Passwort</string>
+    <string name="action_sign_in">Einloggen oder Registrieren</string>
```

Comment:
shouldn't it be either "Einloggen" or "Anmelden" in both cases?

Reply:
Sorry, Can't comment. I am not much knowledgeable and relied on google translate for this. Your call

Reply:
I hoped to not be the only one checking :) both are correct, personally I prefer "Anmelden", but that is just my preference